runsc/main.go

```diff
@@ -82,6 +82,7 @@ var (
 	numNetworkChannels = flag.Int("num-network-channels", 1, "number of underlying channels(FDs) to use for network link endpoints.")
 	rootless           = flag.Bool("rootless", false, "it allows the sandbox to be started with a user that is not root. Sandbox and Gofer processes may run with same privileges as current user.")
 	referenceLeakMode  = flag.String("ref-leak-mode", "disabled", "sets reference leak check mode: disabled (default), log-names, log-traces.")
+	cpuNumFromQuota    = flag.Bool("cpu-num-from-quota", false, "set cpu number to cpu quota (least integer greater than quota value)")
```

Comment:
It's not necessary to add a cmd param for this purpose. We can try to get correct cpu num at Cgroup.NumCPU()?

Reply:
I've added this flag just to make proposed feature opt-in, so users that rely on current behavior are not confused with reduced cpu count after update.

Note that it is a boolean flag that enables the cpu number calculation, not the integer flag with cpu number.

Reply:
I see. And personally, I prefer to make it as default by choosing min(cpus_from_cpuset, cpus_from_cpushare). A lot of softwares are changing into this way to get cpu number.

Reply:
> I prefer to make it as default by choosing min(cpus_from_cpuset, cpus_from_cpushare). A lot of softwares are changing into this way to get cpu number.

Thank you, I've introduced something like `min(cpus_from_cpuset, cpus_from_cpushare, cpus_min)` , where `cpus_min` can be configured.

I'm ok to enable this by default, but it is up to maintainers.

Reply:
It's preferable to keep the default compatible with docker (and cgroups behavior in general). Let's keep it under a flag, maybe remove the cpus_min flag to reduce the changes of someone getting it wrong and to reduce the number of knobs.

Reply:
The cpus_min flag was removed in favor of 2 as a constant. 
This could be easily reverted if needed, currently I can't find a reasonable use-case to change that number.